src/config/webpack.js

```diff
@@ -243,6 +245,7 @@ async function createDevServerConfig(argv, config) {
     noInfo: true,
     overlay: true,
     quiet: true,
+    https: true,
```

Comment:
Can this use the `https` config option that line 65 is using?

Reply:
Not sure I follow. This *is* the default for the option that line 65 uses. It can be overridden in `aunty.config.js` with:
```js
devServer: {
  https: false
}
```

Which would then be used by L65 to generate the correct public path.

At the very least I think we should merge the changes in L65 so it's actually possible to turn https on and still have webpack use the correct `publicPath`.

Reply:
Ah yep, just viewed whole file.